libnetwork/Host.h

```diff
@@ -118,18 +121,54 @@ class Host : public std::enable_shared_from_this<Host>
     /// @return: node id of the connected peer
     std::function<bool(bool, boost::asio::ssl::verify_context&)> newVerifyCallback(
         std::shared_ptr<std::string> nodeIDOut);
+
+    std::string obtainCommonNameFromSubject(std::string const& subject)

```

Comment:
what about move these new function to Host.cpp and also include `#include <boost/algorithm/string/classification.hpp>
#include <boost/algorithm/string/split.hpp>` in Host.cpp?

Reply:
done. please refer to: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/579/files#diff-df0545cce20878ff6751b98f9348901c